### ISSUE TITLE
Pre-trained weights?

### ISSUE THREAD

**Issue by madhawav** [xheon/JointEmbedding#2]
Hi,
Can you provide the pre-trained weights?

**Comment by xheon** [comment]
Hi, sorry for the late reply.

You can find the preprocessed data for both scannet and shapenet together with pretrained weighted here: https://hidrive.ionos.com/share/fg649wmiu5

Please note, that the pre-trained weights refer to the code before this commit (.e. without the 2 sigmoid before the separation model.): https://github.com/xheon/JointEmbedding/commit/4ecbb03d0f0944093b4526077a2df3db61f3b54b

`python test.py --gpu 0 --scannet_path scannet_data_path --shapenet_path shapenet_data_path --confusion_val_path one_to_one_mapping_validation.json --confusion_train_path one_to_one_mapping_train.json --confusion_test_path one_to_one_mapping_test.json --similarity_path scan-cad_similarity_public_augmented-100_v0.json --model_path joint-embedding`

retrieval accuracy: 
```
correct: 2144, total: 5102, accuracy: 0.42022736181889453
02747177:   183/  562 --> 0.326
02808440:    35/   74 --> 0.473
02818832:    91/  165 --> 0.552
02871439:    94/  369 --> 0.255
02933112:    86/  425 --> 0.202
03001627:  1026/ 1828 --> 0.561
03211117:    76/  208 --> 0.365
03337140:    27/  106 --> 0.255
04256520:   100/  265 --> 0.377
04379243:   313/  771 --> 0.406
other:   113/  329 --> 0.343
```

ranking quality
```
correct: 2104, total: 13110, accuracy: 0.1604881769641495
02747177:   141/ 1152 --> 0.122
02808440:    40/  169 --> 0.237
02818832:    81/  429 --> 0.189
02871439:    93/  867 --> 0.107
02933112:    97/ 1010 --> 0.096
03001627:  1016/ 5016 --> 0.203
03211117:    84/  590 --> 0.142
03337140:    25/  251 --> 0.100
04256520:   107/  753 --> 0.142
04379243:   321/ 2099 --> 0.153
other:    99/  774 --> 0.128
```

Best,
Manuel
